File: EP_01/EP_01.2.c
#include <stdio.h>
int main(){
    int numero; 
    scanf("%d",&numero);
    if (numero % 6 == 0){
        int res = 1, saida = 0;
        while (saida != 1){
            if (res * (res-1) *(res-2) == numero){
                printf("SIM\n");
                saida = 1;
            }
            else if (res * (res-1) *(res-2) == numero){
                printf("NAO\n");
                saida = 1;
            }
            res = res + 1;
        }
    }
    else{
        printf("NAO\n");
    }
    return 0;
}

File: EP_01 Introdução/EP_01_2.py
i = int(input("Insira um número:\n"))
if i % 6 == 0:
    n = 1
    out = False
    while not out:
        if n * (n + 1) * (n + 2) == i:
            print("SIM\n")
            out = True
        elif n * (n + 1) * (n + 2) > i:
            print("NAO\n")
            out = True
        n = n + 1
else:
    print("NAO\n")
